suite-native/atoms/src/TransactionItem/TransactionItemWrapper.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { ReactNode } from 'react';
+
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+
+import { VStack } from '../VStack';
+
+type TransactionItemWrapperProps = {
+    children: ReactNode;
+};
+
+const transactionItemWrapperStyle = prepareNativeStyle(utils => ({
+    paddingRight: utils.spacings.medium,
+    paddingLeft: 12,
+    borderRadius: 12,
+}));
+
+export const TransactionItemWrapper = ({ children }: TransactionItemWrapperProps) => {
+    const { applyStyle } = useNativeStyles();
+    return <VStack style={applyStyle(transactionItemWrapperStyle)}>{children}</VStack>;
```

Comment:
Is it necessary to use VStack here? You don't pass any other props here except style... no benefits from VStack. I think that good old `View` can serve here?

Reply:
https://github.com/trezor/trezor-suite/pull/5783/commits/eebfdd500df1a8017caed50b05949391633cbb86